github-orgs/kubeflow/org.yaml

```diff
@@ -636,6 +636,20 @@ orgs:
             privacy: closed
             repos:
               example-seldon: write
+          blog-team:
+            description: Team responsible for maintaining the kubeflow blog
+            maintainers:
+            - jlewi
+            - hamelsmu
+            members:
+            - jbottum
+            - jtfogarty
+            - terrytangyuan
+            - StefanoFioravanzo
+            privacy: closed
+            repos:
+              blog: write
+              fastpages: write
```

Comment:
Should this be removed then?

Reply:
fastpages repo is archived. So one way or the other it doesn't matter much. We can always clean up later.

Reply:
Okay if it’s archived then sounds good.